pkg/deploy/registry/deploylog/rest.go

```diff
@@ -25,8 +26,12 @@ import (
 	deployutil "github.com/openshift/origin/pkg/deploy/util"
 )
 
-// defaultTimeout is the default time to wait for the logs of a deployment
-const defaultTimeout time.Duration = 20 * time.Second
+const (
+	// defaultTimeout is the default time to wait for the logs of a deployment.
+	defaultTimeout time.Duration = 20 * time.Second
```

Comment:
We should return NewServerTimeout

On Jun 29, 2016, at 6:36 AM, Michal Fojtik <notifications@github.com> wrote:

In pkg/deploy/registry/deploylog/rest.go
<https://github.com/openshift/origin/pull/9482#discussion_r68946411>:

> @@ -25,8 +26,12 @@ import (
>  	deployutil "github.com/openshift/origin/pkg/deploy/util"
>  )
>
> -// defaultTimeout is the default time to wait for the logs of a deployment
> -const defaultTimeout time.Duration = 20 * time.Second
> +const (
> +	// defaultTimeout is the default time to wait for the logs of a deployment.
> +	defaultTimeout time.Duration = 20 * time.Second

what error code we return when this timeout is reached? can client retry
getting the logs based on the return code?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/openshift/origin/pull/9482/files/53571d0332a00a9aa0df0515d34a867b65273f42#r68946411>,
or mute the thread
<https://github.com/notifications/unsubscribe/ABG_p6CAHrUsezCmTKGkRaQOC9fxrSlEks5qQnTJgaJpZM4I7sgr>
.